ios/Snapmap/Info.plist

```diff
@@ -34,21 +47,11 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
```

Comment:
need to update orientation for iphone devices above, currently only covers the ipads